.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: status/need-triage, type/bug
+assignees: ''
+
+---
+
+<!--- Provide a general summary of the issue in the Title above -->
+<!--- Make sure to follow the [Contribution Guidelines](https://github.com/reactor/.github/blob/master/CONTRIBUTING.md), -->
+<!--- notably for [security-related issues](https://pivotal.io/security) -->
+<!-- Questions should be asked on [Gitter](https://gitter.im/reactor/reactor) or [StackOverflow](https://stackoverflow.com/questions/tagged/project-reactor). -->
```

Comment:
We have different Gitter rooms per project and also different StackOverflow  tags

Reply:
ok, let's put a link to CONTRIBUTING section about questions instead, and list all the channels there instead

Reply:
Agree

Reply:
done